app/views/ReportView/components/GenomicSummary/components/KeyAlterations/index.tsx

```diff
@@ -114,6 +117,11 @@ const KeyAlterations = ({
           const sorted = sortBy(output, [customTypeSort, 'geneVariant']);
           setVariants(sorted);
           setVariantCounts(counts);
+
+          if (loadedDispatch) {
+            // TODO
```

Comment:
Leftover TODO with no explanation, was this intended?

Reply:
oops not at all let me remove it